hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -122,6 +129,9 @@ import           Hedgehog.Internal.Source
 
 import           Language.Haskell.TH.Lift (deriveLift)
 
+import           Prelude hiding (min)
```

Comment:
😕 What if we rename `min` in `cover min True s` to something else instead hiding the built-in one?